notebooks/community/ml_ops/stage2/get_started_with_vertex_experiments_autologging_tf.ipynb

```diff
@@ -0,0 +1,902 @@
+{
```

Comment:
<p>No variables to setup. I'd suggest to remove the Set up variables comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
done